companies.xml

```diff
@@ -1420,4 +1428,588 @@
+        <country>CH</country>
+    </company>
+    <company>
+        <conid>508626121</conid>
+        <symbol>BTCS</symbol>
+        <name>BTCS Inc</name>
+        <taxNumber>90-1096644</taxNumber>
+        <address>9466 Georgia Avenue, Silver Spring, MD 20910</address>
+        <country>US</country>
+    </company>
+    <company>
+        <conid>5171</conid>
+        <symbol>BP</symbol>
+        <name>BP Plc</name>
+        <taxNumber>00102498</taxNumber>
+        <address>1 St James's Square, London, SW1Y 4PD</address>
+        <country>GB</country>
+    </company>
+    <company>
+        <conid>909054</conid>
+        <symbol>GSK</symbol>
+        <name>GSK Plc</name>
+        <taxNumber>GB239820839</taxNumber>
+        <address>980 Great West Rd, London TW8 9GS</address>
+        <country>GB</country>
+    </company>
+    <company>
+        <conid>574757089</conid>
+        <symbol>GSK</symbol>
+        <name>GSK Plc</name>
+        <taxNumber>GB239820839</taxNumber>
+        <address>980 Great West Rd, London TW8 9GS</address>
+        <country>GB</country>
+    </company>
+    <company>
+        <conid>41957241</conid>
+        <symbol>HOC</symbol>
+        <name>Hochschild Mining Plc</name>
+        <taxNumber>05777693</taxNumber>
+        <address>17 Cavendish Square, London, W1G 0PH</address>
+        <country>GB</country>
+    </company>
+    <company>
+        <conid>240601748</conid>
+        <symbol>INN1d</symbol>
+        <name>ING Groep NV</name>
+        <taxNumber>NL009549420B01</taxNumber>
+        <address>Bijlmerdreef 106, 1102CT Amsterdam</address>
+        <country>NL</country>
+    </company>
+    <company>
+        <conid>11289568</conid>
+        <symbol>NCM</symbol>
+        <name>Newcrest Mining Ltd</name>
+        <taxNumber>20005683625</taxNumber>
+        <address>600 St. Kilda Road, Melbourne, Victoria 3004</address>
+        <country>AU</country>
+    </company>
+    <company>
+        <conid>12150119</conid>
+        <symbol>883</symbol>
+        <name>CNOOC Ltd</name>
+        <taxNumber>0685974</taxNumber>
+        <address>Bank Of China Tower, 1 Garden Rd, Central, Hong Kong</address>
+        <country>HK</country>
+    </company>
+    <company>
+        <conid>119890783</conid>
+        <symbol>O2Dd</symbol>
+        <name>Telefónica Deutschland Holding AG</name>
+        <taxNumber>DE266077779</taxNumber>
+        <address>Georg-Brauchle-Ring 50, 80992 Munich</address>
+        <country>DE</country>
+    </company>
+    <company>
+        <conid>26985516</conid>
+        <symbol>AUY</symbol>
+        <name>Yamana Gold Inc</name>
+        <taxNumber>125079871</taxNumber>
+        <address>40 King Street West, Suite 2100, Toronto ON M5H 3C2</address>
+        <country>CA</country>
+    </company>
+    <company>
+        <conid>10828168</conid>
+        <symbol>PBR</symbol>
+        <name>Petroleo Brasileiro SA</name>
+        <taxNumber>33000167/0245-58</taxNumber>
```

Comment:
According to some Brazilian docs, headquarters always have the 0001-01 ending. This is what I found:
```suggestion
        <taxNumber>33.000.167/0001-01</taxNumber>
```